modules/formulize/formulize_xhr_responder.php

```diff
@@ -329,10 +337,10 @@
 
 }
 
-function renderElement($elementObject, $entryId) {
+function renderElement($elementObject, $entryId, $frid, $screenObject) {
 
 	$GLOBALS['formulize_asynchronousRendering'][$elementObject->getVar('ele_handle')] = true;
-	$deReturnValue = displayElement("", $elementObject, $entryId, false, null, null, false); // false, null, null, false means it's not a noSave element, no screen, no prevEntry data passed in, and do not render the element on screen
+	$deReturnValue = displayElement("", $elementObject, $entryId, false, $screenObject, null, false); // false, null, null, false means it's not a noSave element, no screen, no prevEntry data passed in, and do not render the element on screen
```

Comment:
This is just to play nice, so the render method of the element will receive the screen object, but as mentioned above, that is not actually relevant in any standard elements